### PR TITLE
feat: implement optional credential help actions

### PR DESCRIPTION
**Pull request by bryce-mcmath**
# Summary of Changes

This PR introduces the option of help actions for missing credentials based on cred def ids. The below gif shows three situations; a proof request where the user has the requested credential, a proof request where the user is missing the credential and there is no help action, and a proof request where the credentials are missing and there is help actions.
credential_help_action

# Related Issues

N/A

# Pull Request Checklist

Tick all boxes below to demonstrate that you have completed the respective task. If the item does not apply to your this PR **check it anyway** to make it apparent that there's nothing to do.

- [x] All commits contain a DCO `Signed-off-by` line (we use the [DCO GitHub app](https://github.com/apps/dco) to enforce this);
- [x] Updated LICENSE-3RD-PARTY.md for any added dependencies or vendored components;
- [x] Updated documentation as needed for changed code and new or modified features;
- [x] Added sufficient [tests](../__tests__/) so that overall code coverage is not reduced.

If you have _any_ questions to _any_ of the points above, just **submit and ask**! This checklist is here to _help_ you, not to deter you from contributing!

Pro Tip 🤓

- Read our [contribution guide](../CONTRIBUTING.md) at least once; it will save you a few review cycles!
- Your PR will likely not be reviewed until all the above boxes are checked and all automated tests have passed.

_PR template adapted from the Python attrs project._

### CODE REVIEW

File: packages/legacy/core/App/components/misc/CredentialCard11.tsx [hunk 1]
```diff
@@ -11,12 +13,57 @@ import Icon from 'react-native-vector-icons/MaterialIcons'
...
 import { useTheme } from '../../contexts/theme'
 import { GenericFn } from '../../types/fn'
+import { NotificationStackParams, Screens } from '../../types/navigators'
 import { credentialTextColor, getCredentialIdentifiers, toImageSource } from '../../utils/credential'
 import { formatIfDate, getCredentialConnectionLabel, isDataUrl, pTypeToText } from '../../utils/helpers'
 import { testIdWithKey } from '../../utils/testable'
 
 import CardWatermark from './CardWatermark'
 
+interface CredentialActionFooterProps {
```
Comment (jleach) [inline packages/legacy/core/App/components/misc/CredentialCard11.tsx]: Can this new component be moved to its own file? I feel like this file is getting a little unruly.

File: packages/legacy/core/App/contexts/configuration.tsx [hunk 2]
```diff
@@ -8,6 +8,7 @@ import { EmptyListProps } from '../components/misc/EmptyList'
 import { RecordProps } from '../components/record/Record'
 import { Locales } from '../localization'
 import OnboardingPages from '../screens/OnboardingPages'
+import { HelpEntry } from '../types/help'
```
Comment (jleach) [inline packages/legacy/core/App/contexts/configuration.tsx]: Can this be more descriptive? It's specifically for how to acquire a credential - no?
- GetCredentialHelp
- CredentialInstructions
- etc

File: packages/legacy/core/App/types/help.ts [hunk 3]
```diff
@@ -0,0 +1,10 @@
+import { StackNavigationProp } from '@react-navigation/stack'
```
Comment (jleach) [inline packages/legacy/core/App/types/help.ts]: Same with this file name. "help" is somewhat generic unless its for all app help.

**Review by jleach — CHANGES_REQUESTED** [review]

File: packages/legacy/core/App/components/misc/CredentialCard11.tsx (hunk 1 above)
Comment (bryce-mcmath) [reply packages/legacy/core/App/components/misc/CredentialCard11.tsx]: Done

File: packages/legacy/core/App/contexts/configuration.tsx (hunk 2 above)
Comment (bryce-mcmath) [reply packages/legacy/core/App/contexts/configuration.tsx]: Good point, changed

File: packages/legacy/core/App/types/help.ts (hunk 3 above)
Comment (bryce-mcmath) [reply packages/legacy/core/App/types/help.ts]: Changed

**Review by jleach — APPROVED** [review]

**Pull request merged**